src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoLineWrapCheck.java

```diff
@@ -73,6 +95,129 @@
  *   &lt;property name="tokens" value="IMPORT"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * package com.puppycrawl. // OK
+ * tools.checkstyle.checks;
+ *
+ * import java.io.*; // OK
+ * import java.lang. // violation
+ *  Boolean;
+ *
+ * import static java.math. // OK
+ * BigInteger.ZERO;
+ * </pre>
+ *
+ * <p>
+ * To configure the check to force no line-wrapping only
+ * in class definitions:
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;NoLineWrap&quot;&gt;
+ *   &lt;property name="tokens" value="CLASS_DEF"/&gt;
```

Comment:
Example for class can include method_def and ctor_def as well

Reply:
should i remove dedicated examples of METHOD_DEF and CTOR_DEF in that case or let them be?

Reply:
Yes.
I think it can be one example for class_def, method_def and ctor_def, no need for interface and enum